packages/amplication-build-manager-service/src/queue/queue.service.ts

```diff
@@ -27,4 +45,12 @@ export class QueueService {
     const response = this.kafkaClient.emit(topic, message);
     response.subscribe(observer);
   }
+
+  sendMessage(topic: string, message: string) {
```

Comment:
I believe we should rename this service to `kafkaService`.

The term `queue` is no longer valid when this service consists of emitting events and sending request-response messages. This is simply not a capability of a queueing system.

Reply:
I prefer to do it in another PR, but on the same epic.